install/kubernetes/cilium/templates/cilium-nodeinit/serviceaccount.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if and .Values.agent (not .Values.preflight.enabled) .Values.serviceAccounts.nodeinit.create }}
```

Comment:
should it just be `{{- if .Values.nodeinit.enabled }}` ?

Reply:
Good idea. I copied this statement from the agent. Let me change it.